packages/commerce-sdk-react/src/hooks/sample.test.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React, {useEffect, useState} from 'react'
+import '@testing-library/jest-dom'
+import {screen, waitFor} from '@testing-library/react'
+
+import {mockHttpResponses, renderWithProviders} from '../test-utils'
+import useCommerceApi from './useCommerceApi'
+
+const {withMocks} = mockHttpResponses({directory: `${__dirname}/mock-responses`})
+
+// NOTE: how to easily _update_ the mocked responses:
+// - Globally, via `npm run test:update-mocked-responses`
+// - Per individual test file, by passing in `mode` value -> mockHttpResponses({mode: 'update', directory})
+
+test(
+    'useCommerceApi hook',
+    withMocks(async () => {
+        const Component = () => {
```

Comment:
Can we create the component outside of the test function in case we want to use it in another test? Also I'd make the test function more readable